ui/src/components/Tooltip.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
should move/create it in core-ui because it will likely to be reused

Reply:
Good idea #1366